libsawtooth/src/receipt/store/mod.rs

```diff
@@ -17,7 +17,7 @@
 
 //! Data store for transaction receipts.
 
-#[cfg(feature = "sqlite")]
+#[cfg(any(feature = "postgres", feature = "sqlite"))]
```

Comment:
I think that you can just use `#[cfg(feature = "diesel")]` here

Reply:
that works, updated to just have the diesel feature guard